schematron/v2.1.0/golden_files/L000_Prelim_Analysis.xml

```diff
@@ -0,0 +1,84 @@
+  for preliminary analysis.
+-->
+<auc:BuildingSync xmlns:auc="http://buildingsync.net/schemas/bedes-auc/2019" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://buildingsync.net/schemas/bedes-auc/2019 https://raw.githubusercontent.com/BuildingSync/schema/v2.1.0/BuildingSync.xsd" version="2.1.0">
+  <auc:Facilities>
+    <auc:Facility ID="FacilityA">
+      <auc:Sites>
+        <auc:Site ID="SiteA">
+          <auc:Buildings>
+            <auc:Building ID="BuildingA">
+              <auc:PremisesName>Building Name</auc:PremisesName>
+              <auc:BuildingClassification>Commercial</auc:BuildingClassification>
+              <auc:OccupancyClassification>Office</auc:OccupancyClassification>
+              <auc:YearOfConstruction>2020</auc:YearOfConstruction>
+            </auc:Building>
+          </auc:Buildings>
+        </auc:Site>
+      </auc:Sites>
+
+      <auc:Reports>
+        <auc:Report ID="ReportA">
+          <auc:Scenarios>
+            <auc:Scenario ID="ScenarioA">
+              <!-- 5.2.3.1 -->
+              <auc:ScenarioType>
+                <auc:CurrentBuilding>
+                  <auc:CalculationMethod>
+                    <auc:Measured></auc:Measured>
+                  </auc:CalculationMethod>
+                </auc:CurrentBuilding>
+              </auc:ScenarioType>
+
+              <auc:ResourceUses>
+                <auc:ResourceUse ID="ElectricityA">
+                  <auc:EnergyResource>Electricity</auc:EnergyResource>
+                </auc:ResourceUse>
+              </auc:ResourceUses>
+
+              <auc:AllResourceTotals>
+                <auc:AllResourceTotal ID="AllResourceTotalA">
+                  <!-- 5.2.3.1 -->
+                  <auc:SiteEnergyUseIntensity>123</auc:SiteEnergyUseIntensity>
+                  <!-- 5.2.3.2 -->
+                  <auc:EnergyCostIndex>123</auc:EnergyCostIndex>
+                </auc:AllResourceTotal>
+              </auc:AllResourceTotals>
+
+              <auc:LinkedPremises>
+                <auc:Building>
+                  <auc:LinkedBuildingID IDref="BuildingA"></auc:LinkedBuildingID>
+                </auc:Building>
+              </auc:LinkedPremises>
+            </auc:Scenario>
+
+            <auc:Scenario ID="ScenarioB">
+              <!-- 5.2.3.3 -->
+              <auc:ScenarioType>
+                <auc:Benchmark>
+                  <auc:BenchmarkType>
+                    <auc:PortfolioManager>
+                      <auc:PMBenchmarkDate>2020-01-01</auc:PMBenchmarkDate>
+                    </auc:PortfolioManager>
+                  </auc:BenchmarkType>
+                  <auc:BenchmarkTool>Portfolio Manager</auc:BenchmarkTool>
+                  <auc:BenchmarkYear>2020</auc:BenchmarkYear>
+                  <auc:BenchmarkValue>75</auc:BenchmarkValue>
+                </auc:Benchmark>
+              </auc:ScenarioType>
+              <auc:LinkedPremises>
```

Comment:
One thing I think we should take a stance on is whether one needs to include all parent links to get to a child link (i.e. if you are linking to a Building, one must also link to both the Site and Facility).  Obviously, given the 'lowest' child, one could XPath up to get the `ancestor::Facility` or similar.  In example files previously, this has been done multiple ways:
- ProcessLoad linked only to an `auc:Section`: https://github.com/BuildingSync/schema/blob/bd2741106716208a7f01186578efaca7eb572131/examples/AT_example_SF_audit_report.xml#L2245
- ConveyanceSystem linked to both an `auc:Building` and `auc:Section`: https://github.com/BuildingSync/schema/blob/bd2741106716208a7f01186578efaca7eb572131/examples/AT_example_SF_audit_report.xml#L2255

I think probably the consensus is that we only CHECK for the 'lowest' child necessary in the tree, although others can specify additional.  In that case, this is fine, but wanted to open up for discussion.  Potentially we can add this somewhere to docs.

Reply:
I think linking to one thing could be sufficient for these 211 cases, I agree we should specify which thing it should be linked to.

Reply:
This conversation has been moved to here: https://github.com/BuildingSync/schema/issues/218